server/main.go

```diff
@@ -55,7 +59,10 @@ func getWebHandler(activeFileManager *ActiveFileManager, fileStore FileStore) ht
 			} else if method == "PUT" {
 				// uploading a file
 				handlePutFile(res, req, path[0], activeFileManager)
+			} else if method == "OPTIONS" {
+				return
 			} else {
+				res.Header().Set("Allow", "GET, PUT, OPTIONS")
```

Comment:
>  it's better to follow the spec

That's exactly what I'm suggesting we do. The spec does not say to include Allow header as far as I can tell.